addons/sys_gestures/script_component.hpp

```diff
@@ -0,0 +1,15 @@
+#define COMPONENT sys_gestures
+#define COMPONENT_BEAUTIFIED Radio Gestures
+#include "\idi\acre\addons\main\script_mod.hpp"
+
+#ifdef DEBUG_ENABLED_SYS_GESTURES
+    #define DEBUG_MODE_FULL
+#endif
+
+#ifdef DEBUG_ENABLED_SYS_GESTURES
+    #define DEBUG_SETTINGS DEBUG_SETTINGS_SYS_GESTURES
+#endif
+
+#include "\idi\acre\addons\main\script_macros.hpp"
+
+#define TYPE_BINOCULAR 4096
```

Comment:
We should just copy all of those type defines from ACE3's macros into ACRE2 ones.

Reply:
Probably better in another PR 👀